cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -351,14 +360,16 @@ private String rebalanceStateConditionType(KafkaRebalanceStatus status) {
         RebalanceOptions.RebalanceOptionsBuilder rebalanceOptionsBuilder = convertRebalanceSpecToRebalanceOptions(kafkaRebalance.getSpec());
 
         return computeNextStatus(reconciliation, host, apiClient, kafkaRebalance, currentState, rebalanceAnnotation, rebalanceOptionsBuilder)
-           .compose(desiredStatus -> {
+           .compose(desiredStatusandMap -> {
```

Comment:
Maybe `desiredStatusAndMap` would be better (i.e. uppercase `A` in `and`)?

Reply:
Okay. I have applied that now